### PR TITLE
recalibrate single node performance 

### PR DESCRIPTION
**Pull request by igor-aptos**
## Description
with more datapoints after [this change](https://github.com/aptos-labs/aptos-core/commit/a787263cbc660a75ec4cce88e08a91be32af9a56), we can now recalibrate a little bit more precisely, removing some noise failures.

## Type of Change
- [ ] New feature
- [ ] Bug fix
- [ ] Breaking change
- [ ] Performance improvement
- [ ] Refactoring
- [ ] Dependency update
- [ ] Documentation update
- [ ] Tests

## Which Components or Systems Does This Change Impact?
- [ ] Validator Node
- [ ] Full Node (API, Indexer, etc.)
- [ ] Move/Aptos Virtual Machine
- [ ] Aptos Framework
- [ ] Aptos CLI/SDK
- [ ] Developer Infrastructure
- [ ] Move Compiler
- [ ] Other (specify)

## Checklist
- [ ] I have read and followed the [CONTRIBUTING](https://github.com/aptos-labs/aptos-core/blob/main/CONTRIBUTING.md) doc
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I identified and added all stakeholders and component owners affected by this change as reviewers
- [ ] I tested both happy and unhappy path of the functionality
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

**Review by msmouse — APPROVED** [review]
Thanks boss!

File: testsuite/single_node_performance.py [hunk 1]
```diff
@@ -150,46 +150,46 @@ class RunGroupConfig:
...
-token-v2-ambassador-mint	1	VM	10000	15320.8	16797
-token-v2-ambassador-mint	20	VM	10000	15320.8	16356
-liquidity-pool-swap	1	VM	975	975.7	998
-liquidity-pool-swap	20	VM	8194	8194.6	8497
-liquidity-pool-swap-stable	1	VM	957	957.5	955
-liquidity-pool-swap-stable	20	VM	7881	7881.5	8103
-deserialize-u256	1	VM	10000	33440	37601
-no-op-fee-payer	1	VM	2216	2216.5	2158
-no-op-fee-payer	50	VM	10000	25786.3	27457
+no-op	1	VM	0.822	1.047	38275.3
```
Comment (gelash) [inline testsuite/single_node_performance.py:153]: The comment above still says block size, expected tps, tps.
What's the new format? also how can it work without any other changes in the PR?

Comment (igor-aptos) [reply testsuite/single_node_performance.py:153]: it's the line 148: 
  transaction_type module_working_set_size executor_type min_ratio max_ratio median

other list is if it is calibrated from the logs from the single run

Comment (gelash) [reply testsuite/single_node_performance.py:153]: got it, approving. For my knowledge, where are min_ration and max_ratio taken from? based on min and max of multiple runs?

**Review by gelash — APPROVED** [review]

File: testsuite/single_node_performance.py (hunk 1 above)
Comment (igor-aptos) [reply testsuite/single_node_performance.py:153]: yup

Comment (igor-aptos) [reply testsuite/single_node_performance.py:153]: we don't use it, but we could use that to put a tighter/looser limits based on how variable each test is.

**Pull request merged**